macros/plugins/snowflake/create_external_table.sql

```diff
@@ -29,5 +29,6 @@
     location = {{external.location}} {# stage #}
     {% if external.auto_refresh -%} auto_refresh = {{external.auto_refresh}} {%- endif %}
     {% if external.pattern -%} pattern = '{{external.pattern}}' {%- endif %}
+    {% if external.integration -%} integration = {{external.integration}} {%- endif %}
```

Comment:
Let's quote it, since it's always quoted ([docs](https://docs.snowflake.com/en/sql-reference/sql/create-external-table.html#syntax)):
```suggestion
    {% if external.integration -%} integration = '{{external.integration}}' {%- endif %}
```
Similar to what we have over in `snowflake_create_snowpipe`:
https://github.com/fishtown-analytics/dbt-external-tables/blob/8f7ce3df3ba3eb238d0c759c8b5cd6fa62aa4058/macros/plugins/snowflake/snowpipe/create_snowpipe.sql#L10

Reply:
Done. I might need a day or two to test this locally, but I don't see any issue with this.

Reply:
Hi @jtcohen6 ,
Finally found the time to test the small change locally: It's working fine with the quotes.